slate/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+If this is a question or feature request, make sure to:
```

Comment:
It would be challenging to separate the configuration files and document definition files from the application's core files. Slate's instructions specifically say to make a full copy of the source and put it in your own repository.